client/modules/IDE/components/KeyboardShortcutModal.jsx

```diff
@@ -58,31 +58,31 @@ function KeyboardShortcutModal() {
           <span className="keyboard-shortcut__command">
             {metaKeyName} + S
           </span>
-          <span>{t('Save')}</span>
+          <span>{t('Nav.File.Save')}</span>
```

Comment:
My feeling is that this should be prefixed `KeyboardShortcuts` like everything else? What was your thinking behind this?

Reply:
I have two ideas at the same time:
First, in this way there is only one ‘Save’ mapping across the translation file, so every occurrence of the Save text would be consistent.
If we introduce ‘KeyboardShortcuts.Save’ now we have to check that both translations are consistent.

But also, I suppose that there are parts of the editor that will need to split the same original text  in N different translations, but I don’t know if this entry is one of these cases

Reply:
Is this a good candidate for a top-level prefix called “Common”?

So this could be “Common.Save” and we replace it in both “Nav” and “KeyboardShortcuts”